courses/fundamentals_of_ada/adv_065_primitives.rst

```diff
@@ -27,25 +27,13 @@ Advanced Primitives
 
 .. |checkmark| replace:: :math:`\checkmark`
 
-=================
-Type Derivation
-=================
-
-------------------------------
-Primitive of Multiple Types
-------------------------------
-
-* A subprogram can be a primitive of several types
+============
+Primitives
+============
 
-      .. code:: Ada
-
-         package P is
-            type T1 is range 1 .. 10;
-            type T2 is (A, B, C);
-
-            procedure Proc (V1 : T1; V2 : T2);
-            function "+" (V1 : T1; V2 : T2) return T1;
-         end P;
+-------------
```

Comment:
This (Inheritance) is a slide without content

Reply:
Indeed, I intended to do a recap of primitive, inheritance, type-class, but this is probably too much, removed it